.github/workflows/ci.yml

```diff
@@ -1,16 +1,16 @@
-name: CI
+name: Lint and Test
 
-on: [push]
+on: [pull_request]
```

Comment:
This might need additional triggers if you want to act on GHA-created PRs, not sure what workflows are at play here.